spec/spec_helper.rb

```diff
@@ -99,11 +99,18 @@
     ActionMailer::Base.deliveries = []
   end
 
+  # This is a hack to get around the ssl_required? method in
```

Comment:
Rather than adding an exception for javascript testing could we disable SSL in tests and only enable in the tests that require it? That might make our intention clearer.

i.e. change `ssl_required?` to only return true in production but modify it for the [ssl redirects tests](https://github.com/openaustralia/planningalerts/blob/52d940ccf115724b98101aaa1d04bc691dccd1e3/spec/requests/redirects_spec.rb#L30-L40).